core/src/test/java/dev/streamx/cli/ingestion/IngestionArgumentsValidationTest.java

```diff
@@ -36,8 +33,75 @@ public void shouldRejectChannellessIngestion(QuarkusMainLauncher launcher) {
     LaunchResult result = launcher.launch("unpublish");
 
     // then
-    assertThat(result.exitCode()).isNotZero();
-    assertThat(result.getErrorOutput()).contains("Missing required argument", "<channel>");
+    expectError(result, """
+        Error: Missing required argument(s): (<channel> <key>)
+        Usage: streamx unpublish [-hV] [[--ingestion-url=<propagateIngestionUrl>]]
+                                 (<channel> <key>)
+        Send unpublication trigger
+              <channel>   Channel that message will be published to
+              <key>       Message key
+          -h, --help      Show this help message and exit.
+              --ingestion-url=<propagateIngestionUrl>
+                          Address of 'rest-ingestion-service'
+                            Default: http://localhost:8080
+          -V, --version   Print version information and exit.""");
+  }
+
+  @Test
+  public void shouldRejectIngestionWithMissingParameter(QuarkusMainLauncher launcher) {
+    // when
+    LaunchResult result = launcher.launch("publish", "-s");
```

Comment:
All existing tests are adjusted to use common verification methods (`expectSuccess` / `expectError`) - but this test is actually a new one - representing the scenarion described in https://teamds.atlassian.net/browse/DXP-994